runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -48,7 +48,7 @@ class LayoutFFITypeHelpers
 
 	/**
 	 * @brief Convert the preceding integer of a layout string from string to UDATA,
-	 * and put layout at the position after the integer
+	 * and put layout at the position after the integer.
```

Comment:
Please update the constructor above:
* it can reuse the argument `currentThread` instead of the field `_currentThread`
* the braces should not be on the same line
* remove the extra semicolon
```c
	LayoutFFITypeHelpers(J9VMThread *currentThread)
			: _currentThread(currentThread)
			, _vm(currentThread->javaVM)
	{
	}
```

Reply:
Fixed the format and removed `_currentThread` everywhere (only two places) as `_vm->currentThread` is sufficient in our case.

Reply:
This doesn't seem correct as `currentThread` is not a field of `J9JavaVM`. I will double-check the code to see what we need to do to handle this.

Reply:
> it can reuse the argument currentThread instead of the field _currentThread

Fixed as suggested above.